### ISSUE TITLE
[Bug] v4.0.1 is marked as compatible with versions above 1.21.1 but references class ArmorItem.Type

### ISSUE THREAD

**Issue by NotRyken** [purejosh/pureemeraldtools#2]
**Describe the bug**

When attempting to load the mod on 1.21.2 or higher, it will crash with the following error. This is due to the reference to the `net.minecraft.world.item.ArmorItem.Type` class, which is not valid in versions above 1.21.1

```
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'pureemeraldtools' at 'net.purejosh.pureemeraldtools.PureEmeraldTools'!
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$2(FabricLoaderImpl.java:403)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:401)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at knot//net.minecraft.class_310.<init>(class_310.java:482)
	at knot//net.minecraft.client.main.Main.main(Main.java:248)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:480)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:105)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)
Caused by: java.lang.NoClassDefFoundError: net/minecraft/class_1738$class_8051
	at knot//net.purejosh.pureemeraldtools.PureEmeraldTools.onInitialize(PureEmeraldTools.java:26)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:399)
	... 9 more
Caused by: java.lang.ClassNotFoundException: net.minecraft.class_1738$class_8051
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
	... 11 more
```

**To Reproduce**
1. Create a new instance for 1.21.2 or higher with Fabric loader
2. Install Fabric API and Pure Emerald Tools 4.0.1
3. Launch the instance
4. Observe that the game crashes

**Screenshots**

**Version information**
Fabric/Forge: Fabric
Loader version: 0.16.9
Mod version: 4.0.1
Mod list: fabric-api, pureemeraldtools

**Additional context**

**Comment by purejosh** [comment]
Thanks so much for reporting this @NotRyken, I appreciate it. I was a bit overzealous when selecting the matching versions, I didn't think 1.21.2+ had a change for Armor. I'll revise the compatibility for this mod on their pages, and I'll post a fixed version for 1.21.x when I get around to this mod in my update cycles.

**Comment by purejosh** [comment]
I've changed the versions on all the armor mods I have released to 1.21 and 1.21.1, thanks again. Did you want to leave this open until I release a 1.21.2+ version?

**Comment by NotRyken** [comment]
Awesome, thanks. I think the issue may as well be closed now.
